### PR TITLE
Add AVM methods / Deprecated some APIs

### PR DESCRIPTION
**Pull request by laviniat1996**
This PR adds:

- avm.getBlock
- avm.getBlockByHeight
- avm.getHeight

**Deprecated all ipcs APIs**

- ipcs.publishBlockchain
- ipcs.unpublishBlockchain
- ipcs.getPublishedBlockchains
- Deprecated all keystore APIs
- keystore.createUser
- keystore.deleteUser
- keystore.listUsers
- keystore.importUser
- keystore.exportUser

**Deprecated Avm APIs**

- avm.getAddressTxs
- avm.getBalance
- avm.getAllBalances
- avm.createAsset
- avm.createFixedCapAsset
- avm.createVariableCapAsset
- avm.createNFTAsset
- avm.createAddress
- avm.listAddresses
- avm.exportKey
- avm.importKey
- avm.mint
- avm.sendNFT
- avm.mintNFT
- avm.import
- avm.export
- avm.send
- avm.sendMultiple

**Deprecated platformvm APIs**

- platform.exportKey
- platform.importKey
- platform.getBalance
- platform.createAddress
- platform.listAddresses
- platform.getSubnets
- platform.addValidator
- platform.addDelegator
- platform.addSubnetValidator
- platform.createSubnet
- platform.exportAVAX
- platform.importAVAX
- platform.createBlockchain
- platform.getBlockchains
- platform.getStake
- platform.getMaxStakeAmount
- platform.getRewardUTXOs

### CODE REVIEW

File: Avalanche.postman_collection.json [hunk 1]
```diff
@@ -764,6 +744,126 @@
...
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"jsonrpc\":\"2.0\",\n    \"id\"     :1,\n    \"method\" :\"avm.getBlockByHeight\",\n    \"params\" :{\n        \"height\": {{height}},\n        \"encoding\": \"hex\"\n    }\n}",
```
Comment (cgcardona) [inline Avalanche.postman_collection.json:805]: Please replace `{{height}}` with `{{xhcainBlockHeight}}`

**Review by cgcardona — CHANGES_REQUESTED** [review]
Looks great. I see a minor issue to fix.

File: Avalanche.postman_collection.json (hunk 1 above)
Comment (laviniat1996) [reply Avalanche.postman_collection.json]: fixed ecba6b1451ac334e5b7e55f529e7fd26b0b063e6

Comment (cgcardona) [reply Avalanche.postman_collection.json]: ✅

**Review by cgcardona — APPROVED** [review]

**Pull request merged**